frontend-student/src/components/signup/Signup.jsx

```
@@ -27,149 +47,345 @@ export default function Signup() {
+          </p>
+
+          <div className='space-y-6 mt-12'>
+            <div className='flex items-start space-x-4'>
+              <div className='bg-white bg-opacity-20 p-2 rounded-full'>
+                <svg
+                  xmlns='http://www.w3.org/2000/svg'
+                  className='h-6 w-6'
+                  fill='none'
+                  viewBox='0 0 24 24'
+                  stroke='currentColor'
+                >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)